frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -53,6 +55,17 @@ import {
 } from '../../../ui-components'
 import { CredentialsForm } from '../../Credentials/CredentialsForm'
 import schema from './schema.json'
+import { ExternalLinkAltIcon } from '@patternfly/react-icons'
+import { makeStyles } from '@mui/styles'
+
+const useStyles = makeStyles({
+  select: {
+    alignItems: 'end',
+    '& > div:last-child': {
+      paddingBottom: '6px',
+    },
+  },
+})
```

Comment:
@KevinFCormier your comment on avoiding hard coded styling isn't lost on me just yet, the reason this is here is because it's the same css rule used in the [UpdateAutomationModal](https://github.com/stolostron/console/blob/main/frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/UpdateAutomationModal.tsx#L50-L55). Does that make using it again OK? Or should we refactor?